package.json

```diff
@@ -63,9 +63,6 @@
     "url": "https://github.com/gpbl/react-day-picker/issues"
   },
   "homepage": "https://react-day-picker.js.org",
-  "peerDependencies": {
-    "react": "~0.13.x || ~0.14.x || ^15.0.0"
```

Comment:
Sorry I missed that part 😄 

React [is included](https://github.com/oigewan/react-day-picker/blob/4f6ce4463f60ab3072bc0d5f142a67cb93819f11/package.json#L102) as `devDependencies`, which should be installed when developing the module. So definitely react must stay as `peeDependency` here.